lib/spack/spack/test/gcs_fetch.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+import pytest
+
+import spack.config as spack_config
+import spack.fetch_strategy as spack_fs
+import spack.stage as spack_stage
+
+
+@pytest.mark.parametrize('_fetch_method', ['curl', 'urllib'])
+def test_gcsfetchstrategy_sans_url(_fetch_method):
```

Comment:
Maybe:
```suggestion
def test_gcsfetchstrategy_without_url(_fetch_method):
```
? But in any case :tada: :fr: :tada:

Reply:
👍🏻